stdlib/ephemeron.mli

```diff
@@ -322,15 +324,17 @@ module Kn : sig
       The seed is similar to the one of {!Hashtbl.MakeSeeded}. *)
 
 end
+(** Emphemerons with arbitrary number of keys of the same type. *)
 
 module GenHashTable: sig
   (** Define a hash table on generic containers which have a notion of
       "death" and aliveness. If a binding is dead the hash table can
       automatically remove it. *)
 
   type equal =
-  | ETrue | EFalse
-  | EDead (** the container is dead *)
+  | ETrue
+  | EFalse
+  | EDead
```

Comment:
What's the reason for removing the comment on the constructor here?

Reply:
It appears unprocessed in the manual output. I think this is always the case with these inline comments but, since only one constructor here has a comment (and a seemingly-redundant one), I removed it. Happy to reinstate if others disagree.

Reply:
This is odd, other variant constructors have comments attached to them (in fact many variant declarations use an empty `(**)` comment on the last constructor to avoid getting the following comment attached to only this constructor). cc @Octachron, who may know what is going on.

Reply:
(I am not completely convinced that the documentation comment is redundant. To me it is not obvious what the constructor means, although an explanation is also given in the functions that do use this type. I think it's okay to remove it if it really gets in the way, but it may be worth understanding what is going on first.)

Reply:
It is ocamldoc itself that is purposefully rendering constructor and field comments as if they were non documentation comments (see https://ocaml.org/manual/libref/Gc.html#TYPEstat for a visible example).

Reply:
Thanks. I have re-instated the comment.